tools/upstream-version.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# get latest upstream libyang version from github
+
+RLS_URL=https://api.github.com/repos/CESNET/libyang/releases/latest
```

Comment:
I am not sure this will work. There will definitely be at least one more *libyang* 1 release, which will be marked as latest instead, no?

Reply:
Good point. In that case I recommend updating the script to whatever you think will work... But I don't think it's really a blocker, it's more of a bonus functionality. You can always override autoversion using `-v`/`--version` options.

You can update the script first thing after merging this ;) Or if you have a solution in mind let me know and I'll update the MR once more...

Either way I'd really like to start next week with this already merged if at all possible so that individual issues may be addressed individually.

Reply:
> so that individual issues may be addressed individually.

Right, I just wanted not to merge something we know has an issue already. And the correct fix should probably be to get all the tags and find the highest, no? Should not be too difficult.